src/components/Scrollable/Scrollable.jsx

```diff
@@ -33,10 +33,13 @@ export default class Scrollable extends React.PureComponent {
     constructor(props) {
         super(props);
         this.container = React.createRef();
-        this.vTrack = React.createRef();
-        this.hTrack = React.createRef();
-        this.ctx = {container: this.container};
         this.event = {prev: {}, next: {}};
+        this.state = {
+            cssVarsOnTracks: props.cssVarsOnTracks,
```

Comment:
In the past, we had state and context for the scroll position, and it caused issues (can't recall what they were, but it happened in the Spreadsheet). I would stay away from the React lifecycle for pure CSS things like this and go with lower-level solutions like CSS variables. Especially if the state is going to be updated very often.

That said, if the alternative is to use a ref, context is preferable.
But still, let's see if we can avoid that altogether.

Reply:
now everything is wrapped with `requestAnimationFrame` so the issue you are mentioning might not occur, I will check.

I did all this because of the problem of setting the CSS variables on the parent itself, so now the top & left are set on the tracks (via context & `style`, and not `ref`'s) 

Once (hopefully) Google will fix the Chrome bug I opened for them, I will remove this code because having all variables at the top-level is the best way